### PR TITLE
Feature/form messages and password validation

### PR DESCRIPTION
**Pull request by AshTNA**
Hi @ARBaynes 

Could you check this code please, mainly the regex.

If needed @gtvj can look over it before merging.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,3 +146,13 @@ We have decided to use the following validation rules:
 - at least one number
 
 This follows the validation rules that are currently implemented in Discovery.
+
+## Regex
+
+`(?=.*[a-z].*)(?=.*[A-Z].*)(.*\d.*)`
```
Comment (ARBaynes) [inline README.md]: This seems perfect, although you are missing the 8 char minimum, I've popped it in below:
`(?=.*[a-z].*)(?=.*[A-Z].*)(?=.*\d.*).{8,}`

Comment (ARBaynes) [reply README.md]: Also, this isn't excluding non-alphanumeric characters (even though the rules say that a password is alphanumeric chars only), but that seems like it shouldn't be so much of a problem?

Comment (AshTNA) [reply README.md]: I did not include those in the regex as they are included in the Laravel password string just before the regex:

`required|string|min:8|alpha_num` 

Apologies as I should have mentioned that somewhere in the PR!

Comment (ARBaynes) [reply README.md]: Oooh rad, no worries then! Seems like good regex otherwise 😁

**Pull request merged**